src/components/TableWithBrowserPagination/index.js

```diff
@@ -85,28 +85,38 @@ export default class TableWithBrowserPagination extends Component {
             pageSize,
             data,
             children,
+            variant,
             ...rest
         } = this.props;
         const { activePage, pageItems } = this.state;
         const pages = Math.ceil(data.length / pageSize);
         const showPagination = pages > 1;
+        const paginationVariant = variant === 'listview' ? 'shaded' : 'default';
 
         return (
             <StyledContainer className={className} style={style}>
                 {/* eslint-disable-next-line react/jsx-props-no-spreading */}
-                <Table data={pageItems} {...rest} ref={this.table}>
+                <Table data={pageItems} {...rest} ref={this.table} variant={variant}>
                     {children}
                 </Table>
                 <RenderIf isTrue={showPagination}>
-                    <StyledPaginationContainer paginationAlignment={paginationAlignment}>
+                    <StyledPaginationContainer
+                        paginationAlignment={paginationAlignment}
+                        variant={paginationVariant}
```

Comment:
here instead of the pagination variant, we should pass is the table variant since this makes coupling to the pagination variant when here what matters is the table variant